README.md

```diff
@@ -1,29 +1,43 @@
 # Taxonomy Editor
-Taxonomies are at the heart of Open Food Facts data structure - this project provides an editor
+Taxonomies are at the 🧡 of Open Food Facts data structure which provides an editor.
 
-Docs, see [doc folder](./doc).
+![Github Repo Size](https://img.shields.io/github/repo-size/openfoodfacts/taxonomy-editor?style=for-the-badge&color=aqua)
 
-> **NOTE:**
-> This documentation tries to follow as much as possible the documentation system from [Diátaxis](https://diataxis.fr/).
 
-## Contribute
+Visit the [doc folder](./doc)
+> This documentation tries to follow as much as possible the documentation system from [Diataxis](https://diataxis.fr/).
 
-- to develop: [Setup a dev environment](./doc/introduction/setup-dev.md)
-- to translate: use [Crowdin](https://crowdin.com/project/openfoodfacts), project open food facts
+## Contributing
+
+- <b>to develop:</b> [Setup a dev environment](./doc/introduction/setup-dev.md)
+- <b>to translate:</b> use [Crowdin](https://crowdin.com/project/openfoodfacts), project open food facts
 - report issues, feature requests etc.
 
 
-Wiki Page: https://wiki.openfoodfacts.org/GSOC_2022_-_Taxonomy_editor
 
-GitHub project: https://github.com/orgs/openfoodfacts/projects/28/views/1
+## Links
+
+<p>
```

Comment:
@utkarsh006 Please try to use the Markdown syntax everywhere and use HTML tags in places you'd need granular control.